src/main/java/com/solvd/db/service/AccountTransactionService.java

```diff
@@ -0,0 +1,25 @@
+package com.solvd.db.service;
+
+import com.solvd.db.customexception.DAONotFoundException;
+import com.solvd.db.dao.interfaces.IAccountTransactionDAO;
+import com.solvd.db.dao.mysql.AccountTransactionDAO;
+import com.solvd.db.factory.JdbcDAOFactory;
+import com.solvd.db.model.AccountTransaction;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AccountTransactionService {
+    private JdbcDAOFactory jdbcDAOFactory = new JdbcDAOFactory();
+
+    public List<AccountTransaction> getAccountTransactionsByAccountNumber(int accountNumber) {
+        try {
+            IAccountTransactionDAO accountTransactionDAO = (AccountTransactionDAO) jdbcDAOFactory.getDAO(AccountTransaction.class.getSimpleName());
+            return accountTransactionDAO.getTransactionByAccountNumber(accountNumber);
+        } catch (DAONotFoundException e) {
+            System.out.println(e);
+        }
+        return new ArrayList<>();
```

Comment:
to be more readable, I recommend naming the empty arrayList first , assign it with the operation in the  return statement in the try block, and return the arraylist at the end.